src/objects_on_table_vis.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
```

Comment:
I understand this is easier for testing, but I think it would be more useful to have a visualization function instead, that could be turned on or off using a parameter that would be called from the GetObjectOnTable service instead of a separate component that continuously calls the service.

In normal use, we would want to be certain that what we see is indeed the bounding box that were returned by the service as called by the pipeline, and keep those visualized until the next call, instead of something that continuously call the service independently

Reply:
The visualization function that is used for rviz is currently implemented in the grasping pipeline (and only called once for each grasp). Should I move that function to this repository?

Reply:
Oh good point ... I would still have the visualization you made in this package available as described. The grasping pipeline can make its own visualization according to its own needs but if this package is used independently it would still be nice to have it available.
This is a good time to introduce `pub.get_num_connections()` to avoid unnecessary computation of the publisher when there is no subscriber!

Reply:
Implemented.
Moved the visualization function to this repository. Visualization can be toggled with rosparam.